src/components/vue/update-explorer/Explorer.vue

```diff
@@ -156,10 +157,14 @@ onMounted(async () => {
       v-if="error"
       class="fixed inset-0 z-50 flex h-screen w-screen items-center justify-center overflow-y-auto bg-gray-600 bg-opacity-50"
     >
-      <div class="w-96 rounded-md border-l-8 border-l-red-500 bg-altbackground p-5 shadow-lg">
+      <div class="max-w-min rounded-md border-l-8 border-l-red-500 bg-altbackground p-5 shadow-lg">
         <span class="text-lg font-bold">An error has occured</span>
         <p class="text-sm">There was an unexpected error. Check the details below:</p>
-        <pre class="mt-3 bg-background">{{ JSON.stringify(error, null, 2) }}</pre>
+        <pre class="mt-3 break-all bg-background" v-if="error.stack?.includes('getMe')">
```

Comment:
Too broad

Reply:
Broad as in?

Reply:
includes getMe

Reply:
I know 
I mentioned that it's hacky because as of now there's no way to detect of the error is from `getMe` method, blso either we use a `.catch` with the getMe method and catch the errors of that method seperately or do some hacks.